### PR TITLE
Add missing localized text

### PR DESCRIPTION
**Pull request by cinqmilleans**
The text DetailsViewHeaderFlyout_ShowDateCreated.Text is not localized. This pr fix that.
The spaces at the end of lines have been removed in the modified file, which explains the large number of modified lines.

### CODE REVIEW

File: Files/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -2860,4 +2860,7 @@ We use App Center to keep track of app usage, find bugs, and fix crashes. All in
   <data name="SideBarFavoritesMoveToTop" xml:space="preserve">
     <value>Move to top</value>
   </data>
+    <data name="DetailsViewHeaderFlyout_ShowDateCreated.Text" xml:space="preserve">
```
Comment (BanCrash) [inline Files/Strings/en-US/Resources.resw:2863]:
```suggestion
  <data name="DetailsViewHeaderFlyout_ShowDateCreated.Text" xml:space="preserve">
```

Comment (BanCrash) [reply Files/Strings/en-US/Resources.resw:2863]: I didn't reviewed it yet, but this indentation is wrong ;P

**Comment by yaichenbaum** [comment]
@cinqmilleans can you resolve the merge conflicts?

**Comment by cinqmilleans** [comment]
@yair100 It’s done.

**Comment by BanCrash** [comment]
@cinqmilleans do you test this with some xlf file? For some reason my VS is not adding the new string to the XLF files, and adding it manually didn't work neither. Since I'm using a virtual machine maybe it's not working as expected, that's why I want to confirm.

**Comment by BanCrash** [comment]
> @BanCrash I just tested, it works for me. You must install "Multilingual App Toolkit" (VS extension) for the files to be regenerated.

Thanks for the tip, I didn't remember to do that 🤕

File: Files/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -2767,4 +2767,7 @@ We use App Center to keep track of app usage, find bugs, and fix crashes. All in
   <data name="SideBarFavoritesMoveToTop" xml:space="preserve">
     <value>Move to top</value>
   </data>
+  <data name="DetailsViewHeaderFlyout_ShowDateCreated.Text" xml:space="preserve">
+    <value>Date created column</value>
+  </data>
```
Comment (BanCrash) [inline Files/Strings/en-US/Resources.resw:2772]: Your code is good, but I think we can use this PR to add the other remaining string on the same flyout:

```suggestion
  </data>
  <data name="DetailsViewHeaderFlyout_ShowItemSize.Text" xml:space="preserve">
    <value>Item size column</value>
  </data>
```

**Comment by cinqmilleans** [comment]
@BanCrash It's done. Thank you.

**Review by BanCrash — APPROVED** [review]
Perfect, thank you!

**Pull request merged**